types/koa-send/index.d.ts

```diff
@@ -29,7 +29,7 @@ declare namespace send {
         /** Root directory to restrict file access. (defaults to '') */
         root?: string;
         /** Name of the index file to serve automatically when visiting the root location. (defaults to none) */
-        index?: string;
+        index?: string | false;
```

Comment:
Why `false` here? I don't see how the behavior of koa-send would change if `index` were `false`.

Reply:
That's true as `''` (empty string) is a falsely value, and  for `koa-send` that is enough (https://github.com/koajs/send/blob/master/index.js#L66 https://github.com/koajs/send/blob/master/index.js#L111).

But `koa-static` uses comparison to `false` (https://github.com/koajs/static/blob/master/index.js#L36), so I'd like to have your advise in this case @pe8ter.
Maybe also other options (eg. `root` - https://github.com/koajs/send/blob/master/index.js#L43) should be able to accept falsely types (`void`, `false`, `number` - I'm not sure with this one).

Reply:
Would it be possible to make this PR without this change and then re-open the question in a separate PR? Everything else looks reasonable to me.

Reply:
@pe8ter can reply to the question by @tlaziuk.
@tlaziuk can you create a separate PR for that change... so we can get the rest in?

Reply:
I now disagree with my earlier objection. One should explicitly use `false` to disable the default _index.html_.